src/mz/src/regions.rs

```diff
@@ -177,3 +166,85 @@ pub(crate) async fn list_cloud_providers(
         .json::<Vec<CloudProvider>>()
         .await
 }
+
+/**
+ * Prints if a region is enabled or not
+ * E.g.: AWS/us-east-1  enabled
+ */
+pub(crate) fn print_region_enabled(cloud_provider_and_region: &CloudProviderAndRegion) {
+    let region = &cloud_provider_and_region.region;
+    let cloud_provider = &cloud_provider_and_region.cloud_provider;
+
+    match region {
+        Some(_) => println!(
+            "{:}/{:}  enabled",
+            cloud_provider.provider, cloud_provider.region
+        ),
+        None => println!(
+            "{:}/{:}  disabled",
+            cloud_provider.provider, cloud_provider.region
+        ),
+    };
+}
+
+/**
+ * Prints a region's status and addresses
+ * Healthy:         {yes/no}
+ * SQL address:     foo.materialize.cloud:6875
+ * HTTPS address:   <https://foo.materialize.cloud>
+ */
+pub(crate) fn print_region_status(region: Region, health: bool) {
+    if health {
+        println!("Healthy:\tyes");
+    } else {
+        println!("Healthy:\tno");
+    }
+    println!("SQL address: \t{}", region.environmentd_pgwire_address);
+    // Remove port from url
+    println!(
+        "HTTPS address: \thttps://{}",
+        &region.environmentd_https_address[0..region.environmentd_https_address.len() - 4]
+    );
+}
+
+// ------------------------------------------------------------------------
+// Delete is currently disabled. Preserving the code for once is available.
```

Comment:
```suggestion
// Delete is currently disabled. Preserving the code for once is available.
```

Delete is actually always going to be disabled (it'll be a support-only operation), so I think we should just pull the plug on this now.